contracts/LinniaPolicies.sol

```diff
@@ -0,0 +1,110 @@
+pragma solidity 0.4.24;
+
+import "openzeppelin-solidity/contracts/lifecycle/Destructible.sol";
+import "openzeppelin-solidity/contracts/lifecycle/Pausable.sol";
+
+import "./LinniaHub.sol";
+
+import "./interfaces/PolicyI.sol";
+
+/**
+
+    This contract acts as a container for all policy related logic in Stow protocol.
+
+    A policy is a smart contract that looks at the state of a record or a permission
+    and validates whether certain logic is true. Policies are used when creating records
+    and granting access to records to make sure that certain contraints are being met.
+    Users can choose to add and remove constraints their own records. App creators can
+    also choose to conform to policies while faciliating the permissioning of their user's
+    files.
+
+**/
+
+contract LinniaPolicies is Pausable, Destructible {
+
+    LinniaHub public hub;
+
+    event LinniaPolicyChecked(
+        bytes32 indexed dataHash,
+        string dataUri,
+        address indexed viewer,
+        address indexed policy,
+        bool isOk,
+        address sender
+    );
+
+    /* Constructor */
+    constructor(LinniaHub _hub) public {
+        hub = _hub;
+    }
+
+    /* Fallback function */
+    function () public { }
+
+    modifier onlyUser() {
+        require(hub.usersContract().isUser(msg.sender) == true);
+        _;
+    }
+
+    /* @dex checks an array of policies to make sure they are valid */
+    /* @param dataHash the dataHash of the record being checked */
+    /* @param viewer the address of the person being given access the record */
+    /* @param dataUri where the record is */
+    /* @param policies an array of the policy conforming addresses */
+    function policiesAreValid(
```

Comment:
I think this should be `internal`

Reply:
People may want to reveal to people which policies their records and permissions can/will conform with from a UI. Keeping public also allows people to surface better error messages from the UI